### ISSUE TITLE
request for a `-u USER` option

### ISSUE THREAD

**Issue by plesager** [plesager/ece3-postproc#13]
@oloapinivad wrote in https://github.com/plesager/ece3-postproc/pull/11#issuecomment-369208141

>  So we would like to change the IFSRESULTS0 and NEMORESULTS0 but only at user level. Probably it is enough to do something like -> if a -u USERexp is specified, apply a sed on the $USER inside the IFSRESULTS0 and NEMORESULTS0 string replacing with the USERexp, without the need of defining an env variable USERexp. In theory, this can work in parallel with ALTDIR, but I am agree that ALTDIR is not very useful in the very end.
If you agree, I will implement this in the next days.

**Comment by plesager** [comment]
Yes go ahead with (1) removing ALTDIR in hiresclim only (may still be useful in ecmean?) and (2) with adding the -u USER option (in _hc.sh_ and _master_hiresclim.sh_) to define a USERexp at the command line.

But I am not 100% sold on using `sed` to handle it, when compared to using an env variable in the config file only: `export ${USERexp:=$USER}`.
What if somebody wants to change something else than `$USER`? With the sed solution, you have to edit the `sed` command in _master_hiresclim.sh_ at the risk of breaking it. With the en variable, you can just decide what USERexp would replace by just changing your config to `export ${USERexp:=ANOTHERDEFAULT}`. This is way more flexible. Maybe the USERexp flag needs another easy-to-remember name then: alternate, substitute, replacement, token... none as easy to remember than a "-u USER"?
On the other hand, this "what if" is very hypothetical, a niche case. So if you prefer and want to go ahead with a `sed` call, there is no problem.

**Comment by oloapinivad** [comment]
Hi, I introduced the -u USER using the env variable as you suggested, it seems working fine and we are testing it in https://github.com/oloapinivad/ece3-postproc/commit/9b786d3e3674afd8b1924c26f1237fb61bd9ba1d.

For the moment we kept the `export ${USERexp:=$USER}` structure but we can further generalize it later (for different replacements).  Once we are sure it is working I will do a pull request. Thanks for the help.
